packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1789,13 +1789,6 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   /// Returns `false` if a toolbar couldn't be shown, such as when the toolbar
   /// is already shown, or when no text selection currently exists.
   bool showToolbar() {
-    // Web is using native dom elements to enable clipboard functionality of the
-    // toolbar: copy, paste, select, cut. It might also provide additional
-    // functionality depending on the browser (such as translate). Due to this
-    // we should not show a Flutter toolbar for the editable text elements.
```

Comment:
We are not planning to show toolbar on web. 

Toolbar will be shown by mobile web browsers natively when the user is interacting with an editable text. Desktop web browsers does not show toolbar.

In other words we are mostly trying to optimize a natural interaction for all platforms (and browsers)

Reply:
Noted